.github/workflows/rubocop.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Install gems
         run: |
-          gem install rubocop -v 0.91.0
+          gem install rubocop -v 1.7.0
```

Comment:
This syncs the rubocop version used in the GHA workflow with the one defined in the Gemfile.lock file